subsys/profiling/Kconfig

```
@@ -0,0 +1,14 @@
+# cOPYRIGHT (C) 2023 kns gROUP llc (yadro)
+#
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig PROFILING
+	bool "Profiling tools"
+	help
+		Enable profiling tools, such as perf
```

This and other Kconfig files, fix indent
```suggestion
	  Enable profiling tools, such as perf
```

---

Thanks for the comment. Corrected it.